controllers/llamaCPP.h

```diff
@@ -1,7 +1,11 @@
+#define NOMINMAX
```

Comment:
is this windows specific

Reply:
Yes, if do not add it, Windows cannot build

Reply:
I think I should add it to a condition